### ISSUE TITLE
Wrong headline structure (WCAG 1.3.1a)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1219]
Headings that semantically belong to levels h1 and h2 are marked as levels h3 and h4.

https://slidewiki.org/presentation/81127-1/overview-of-accessibility/81127-1/437649-3/?language=en-/slide-719796-3

**Comment by aoelen** [comment]
Unfortunately we cannot control the headings of the slide content (created by users). This isn't something we can easily change. If you have any suggestions for this, I can take a look. Otherwise I suggest to close this issue.

**Comment by VivienDZB** [comment]
I know, you can't control that. But it should be at least possible to change it to Heading 1 oder 2 if wanted. Right now there's no option for it in the menu:

![grafik](https://user-images.githubusercontent.com/48785136/81389343-f6db5280-9119-11ea-8d11-0fd6be9063d8.png)

Perhaps it would also make sense to integrate a note for the user at some point that changing the heading hierarchy is necessary/useful in the sense of accessibility.
